### PR TITLE
Added command for testing all apps  inside apps_dev, dummy unit tests for examples apps.

### PR DESCRIPTION
**Pull request by rbharadwaj9**
### Command: `python manage.py test_nd_apps`
This command is the same as `python manage.py test` but in case a path is not provided, it looks for apps inside `apps_dev`

### CODE REVIEW

File: src/netdash/netdash_ui/management/commands/test_nd_apps.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.core.management.utils import get_command_line_option
+from django.test.utils import get_runner
+
+
+class Command(BaseCommand):
+    help = 'Discover and run tests in the specified modules or the current directory.'
```
Comment (jlost) [inline src/netdash/netdash_ui/management/commands/test_nd_apps.py]: You should update the help text so it's clear how this is different from the normal `test` command.

File: src/netdash/netdash_ui/management/commands/test_nd_apps.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.core.management.utils import get_command_line_option
+from django.test.utils import get_runner
+
+
+class Command(BaseCommand):
```
Comment (jlost) [inline src/netdash/netdash_ui/management/commands/test_nd_apps.py]: If you inherit from the original Django `test` `Command`, you won't need to copypaste `run_from_argv` and `add_arguments`

File: src/netdash/netdash_ui/management/commands/test_nd_apps.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+        if hasattr(test_runner_class, 'add_arguments'):
+            test_runner_class.add_arguments(parser)
+
+    def handle(self, *test_labels, **options):
+        TestRunner = get_runner(settings, options['testrunner'])
+
+        test_runner = TestRunner(**options)
+        if not test_labels:
+            test_labels = ["../../apps_dev/", ]
```
Comment (jlost) [inline src/netdash/netdash_ui/management/commands/test_nd_apps.py]: I take it these two lines are the only deviation from the original? You could do these two lines, and then call the `super` version of `handle` with the new `test_labels` if you inherit from the original Django `test`.

Comment (rbharadwaj9) [reply src/netdash/netdash_ui/management/commands/test_nd_apps.py]: Sounds  good I'll look into it

**Comment by jlost** [comment]
Very nice!

File: src/netdash/netdash_ui/management/commands/test_nd_apps.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+from django.core.management.commands import test
+
+
+class Command(test.Command):
+    help = 'Discover and run tests in the specifiied modules or in the apps_dev directory'
+
+    def run_from_argv(self, argv):
+        super(Command, self).run_from_argv(argv)
+
+    def add_arguments(self, parser):
```
Comment (jlost) [inline src/netdash/netdash_ui/management/commands/test_nd_apps.py]: @rbharadwaj9 I noticed this after merging, but if you extend a class, you get all of its methods by default, so you don't need to reimplement these two methods with calls to `super`.

Comment (rbharadwaj9) [reply src/netdash/netdash_ui/management/commands/test_nd_apps.py]: That is what I assumed but I couldn't get it to work that way. I will look into it more and make the fixes appropriately. However, this is how I saw two other pages do it.

**Pull request merged**